examples/fodo.py

```diff
@@ -1,11 +1,6 @@
 import json
-import os
-import sys
 import yaml
 
-# Add the parent directory to sys.path
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../src/")))
-
```

Comment:
Good that this is not needed - I don't remember why I had added it in the first place.

Reply:
Sure! Probably just from an early draft or so running the script directly without `pip install -e .`